editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx

```diff
@@ -243,14 +255,13 @@ const RemixSceneLabel = React.memo<RemixSceneLabelProps>((props) => {
         className='roleComponentName'
         style={{
           pointerEvents: labelSelectable ? 'initial' : 'none',
-          color: isIndexRoute ? colorTheme.aqua.value : colorTheme.textColor.value,
+          color: colorTheme.aqua.value,
           position: 'absolute',
           left: frame.x,
           bottom: -frame.y + offsetY,
           width: frame.width,
-          paddingLeft: offsetX,
-          paddingTop: paddingY,
-          paddingBottom: paddingY,
+          height: scaledLineHeight * 2,
```

Comment:
visually the new bars are still much too tall and prominent for me. Can we bring this down to something more like 2-3px top/bottom padding around the text?